.scalafmt.conf

```diff
@@ -1,4 +1,23 @@
 version=2.4.2
 
-align = true    # For pretty alignment.
+align = more    # For pretty alignment.
 maxColumn = 140 # For my wide 30" display.
+
+rewrite.rules = [
+  SortModifiers
+  RedundantBraces
+  RedundantParens
+  AsciiSortImports
+  PreferCurlyFors
+]
+
+rewrite.redundantBraces.stringInterpolation = true
+rewrite.redundantBraces.includeUnitMethods = true
+
+newlines.afterCurlyLambda = preserve
```

Comment:
This is what allows [this format](https://github.com/laserdisc-io/laserdisc/compare/master...barambani:formatting?expand=1#diff-593e74efa3545c9a9d1ce565416716c1R42). If you set it like that it will leave it in peace.